notebooks/end-to-end-structured/solutions/4b_keras_dnn_babyweight.ipynb

```diff
@@ -11,7 +11,6 @@
     "1. Set CSV Columns, label column, and column defaults\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.        # Create two hidden layers of [64, 32] just in like the BQML DNN
</pre><p>remove mention of BQML DNN</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/204/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>